docker-compose-v2.yml

```diff
@@ -225,3 +222,7 @@ services:
       - HOTEL_URL=http://hotel:9101
       - CAR_URL=http://car-rental:9102
       - CURRENCY_EXCHANGE_URL=http://currency-exchange:9201
+      - FLIGHTS_URL=http://flights:9103
```

Comment:
The ui should depend on flights and I get this error for the ui when running `docker-compose up` with this file: `ERROR: Service 'ui' failed to build: The command '/bin/sh -c yarn build' returned a non-zero code: 1`